source/common/network/address_impl.h

```diff
@@ -312,6 +330,14 @@ class PipeInstance : public InstanceBase {
   }
   absl::string_view addressType() const override { return "default"; }
 
+  void abslHashValue(absl::HashState state) const override {
```

Comment:
PipeInstance::operator== is based on asString(). Can you document why we know that equal PipeInstances will have equal HashStates and unequal PipeInstances will have unequal HashStates?

Reply:
I'm wrong here, I should have a different `operator==` here with different method name here for the hash.